src/Nethermind/Nethermind.Init/Steps/StartMonitoring.cs

```diff
@@ -16,7 +16,7 @@
 
 namespace Nethermind.Init.Steps;
 
-[RunnerStepDependencies(typeof(InitializeNetwork))]
+[RunnerStepDependencies()]
```

Comment:
It doesn't need anything to run before it?

Reply:
Sorry, my bad. Needed to remove it or the metric does not show up so I can't measure perf. I'm not sure if it actually need it or not though.